vpr/src/base/vpr_context.h

```diff
@@ -145,6 +146,12 @@ struct DeviceContext : public Context {
     /*
      * Structures to define the routing architecture of the FPGA.
      */
+    RRGraphView rr_graph = RRGraphView(rr_nodes, rr_spatial_lookup); // A read-only view of routing resource graph to be the ONLY database for client functions: GUI, placer, router, timing analyzer etc.
```

Comment:
Make sense to me. I have moved the ``RRGraphView`` and ``RRSpatialLookup`` after the ``rr_node_indices``, so that they create references only after the dependent datastuctures have been initalized.